apps/vr-tests/src/stories/ButtonNext.stories.tsx

```diff
@@ -14,6 +14,9 @@ storiesOf('Button Next', module)
   .addDecorator(story => (
     <Screener
       steps={new Steps()
+        .executeScript(
+          'const script=document.createElement(\'script\');script.src="https://cdn.jsdelivr.net/gh/nuxodin/ie11CustomProperties@4.1.0/ie11CustomProperties.min.js";document.head.appendChild(script);location.reload();',
```

Comment:
why do you need to reload? the script will execute before we render the button. that way the button should have the styles

Reply:
actually, steps are executed after stories are rendered. so this will not work :) we need a way to inject and execute the script before the stories render

Reply:
https://storybook.js.org/docs/react/configure/story-rendering#adding-to-head this might be what we want